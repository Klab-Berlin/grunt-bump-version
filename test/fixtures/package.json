{
  "name": "test",
  "description": "test",
  "version": "0.0.0",
  "author": "Frederik Rudeck <frederik.rudeck@googlemail.com>"
}
