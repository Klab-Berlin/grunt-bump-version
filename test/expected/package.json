{
  "name": "test",
  "description": "test",
  "version": "0.0.1",
  "author": "Frederik Rudeck <frederik.rudeck@googlemail.com>"
}
